### PR TITLE
applications: nrf5340_audio: Update controller to 3357

### PR DESCRIPTION
**Pull request by rick1082**
Update audio subsystem controller to build 3357.
OCT-2606
Signed-off-by: Jui-Chou Chung <jui-chou.chung@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst [hunk 1]
```diff
@@ -9,6 +9,19 @@ Changelog
...
 
+Controller v3357
+****************
+
+The following changes have been introduced in the v3357 of the controller:
+
+Changes
+=======
+
+* Fixed an issue where P0.28 - P0.31 cannot be allocated for controlling FEM.
```
Comment (greg-fer) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:20]:
```suggestion
* Fixed an issue where pins P0.28 to P0.31 cannot be allocated for controlling front-end modules (FEMs).
```

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst [hunk 2]
```diff
@@ -9,6 +9,19 @@ Changelog
...
+* Fixed an issue where the FEM control pins cannot work when TX output power setting is 0 dBm.
```
Comment (greg-fer) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:21]:
```suggestion
* Fixed an issue where the FEM control pins cannot work when the TX output power setting equals 0 dBm.
```

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst [hunk 3]
```diff
@@ -9,6 +9,19 @@ Changelog
...
+* Be able to read RSSI from CIS ISO channel through HCI_Read_RSSI HCI command.
```
Comment (greg-fer) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:22]:
```suggestion
* Updated the controller to be able to read RSSI from the CIS ISO channel using the ``HCI_Read_RSSI`` HCI command.
```
This should go to the line 20, that is above the fixes that are currently in lines 20-21. We always list updates before bug fixes.

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst [hunk 4]
```diff
@@ -9,6 +9,19 @@ Changelog
...
+Controller v3357
+****************
+
+The following changes have been introduced in the v3357 of the controller:
+
+Changes
+=======
+
+* Updated the controller to be able to read RSSI from the CIS ISO channel using the ``HCI_Read_RSSI`` HCI command.
+* Fixed an issue where pins P0.28 to P0.31 cannot be allocated for controlling front-end modules (FEMs).
```
Comment (alexsven) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:21]: This is mentioned in the known issues in doc/nrf/known_issues.rst, so it should be removed from there in this PR

**Review by alexsven — CHANGES_REQUESTED** [review]

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst (hunk 4 above)
Comment (rick1082) [reply doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:21]: hmmmm, that is attached with a specific tag release, so probably the proper way is making the issue stick with (and just stop at) tag v2.4.0.

File: lib/bin/bt_ll_acs_nrf53/bin/manifest.yaml [hunk 5]
```diff
@@ -1,7 +1,7 @@
...
-git_revision: fc7d0ad36be2097ba378d12da0c4b8a0bed0138a
-ll_subversion_number: '0x0D15'
+git_revision: b159297b0eae18c6af0aedc46943b6248df6b6e2
+ll_subversion_number: '0x0D1D'
 ll_version_number: '0x0B'
-ll_subversion_number_decimal: '3349'
+ll_subversion_number_decimal: '3357'
 ll_version_number_decimal: '11'
-timestamp: '2023-04-28T11:20:32Z'
+timestamp: '2023-06-06T08:06:07Z'
```
Comment (koffes) [inline lib/bin/bt_ll_acs_nrf53/bin/manifest.yaml:7]: This timestamp: 06-06, is this correct? @Frodevan can you comment?

**Review by koffes — CHANGES_REQUESTED** [review]

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst (hunk 4 above)
Comment (alexsven) [reply doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:21]: Oh yeah, you're right. Sorry about that

**Review by alexsven — APPROVED** [review]

File: lib/bin/bt_ll_acs_nrf53/bin/manifest.yaml (hunk 5 above)
Comment (rick1082) [reply lib/bin/bt_ll_acs_nrf53/bin/manifest.yaml:7]: Time flies, that's the date when we triggered the build.

**Review by Frodevan — APPROVED** [review]

**Review by koffes — APPROVED** [review]
Approved, the checklist needs to be finished though.

**Review by greg-fer — APPROVED** [review]

**Pull request merged**